app/ide-desktop/lib/dashboard/esbuild-config.ts

```diff
@@ -107,7 +107,9 @@ export function bundlerOptions(args: Arguments) {
             // The `dataurl` loader replaces the import with the file, as a data URL. Using the
             // `file` loader, which copies the file and replaces the import with the path.
             /* eslint-disable @typescript-eslint/naming-convention */
-            esbuildPluginInlineImage({ extensions: ['svg'] }),
+            esbuildPluginInlineImage({
+                extensions: ['svg'],
+            }) as esbuild.Plugin,
```

Comment:
That change seems unnecessary. If you have type errors, double-check if you didn't install two different esbuild versions.